client/components/BudgetGrid/style.scss

```diff
@@ -30,6 +30,11 @@
 
     &:last-child {
       text-align: right;
+
+      button {
```

Comment:
if `button` is directly nested under budgetgrid component than keep here. But if you use button somewhere else (e.g. EntryForm) then let's move this css to a file just for that component